docs/userguide/unicode.md

```diff
@@ -138,7 +138,7 @@ more complexity into the Unicode standard, but far less than managing a large
 number of different encodings.
 
 Starting with Unicode 2.0 (published in 1996), the Unicode standard began
-assigning numbers from 0 to 10ffff16, which requires 21 bits but does not use
+assigning numbers from 0 to 10ffff₁₆, which requires 21 bits but does not use
```

Comment:
Just as a FYI: You can do `<sup>16</sup>` in Markdown as well.

Reply:
Awesome. I also noticed `<sub>16</sub>` can be used, which is the case. Replacing all references with this approach.